tests/PySys/c8y_restart_bridge/run.py

```diff
@@ -0,0 +1,45 @@
+import pysys
+from pysys.constants import *
+from pysys.basetest import BaseTest
+
+import sys
+
+sys.path.append("environments")
+from environment_c8y import EnvironmentC8y
+
+"""
+Bridge restart
+
+Given a configured system with configured certificate
+When we setup EnvironmentC8y
+When we validate EnvironmentC8y
+When we cleanup EnvironmentC8y
+Then then the test has passed
+
+So far just use the EnvironmentC8y nothing else:
+see ../environment/environment_c8y
+"""
+
+
+class PySysTest(EnvironmentC8y):
+    def setup(self):
+        super().setup()
+        self.log.info("Setup")
+        self.addCleanupFunction(self.mycleanup)
+
+    def execute(self):
```

Comment:
This explicit redefinition of `execute` method is not really useful, right?

Reply:
I kept the execute to stick to the four phase test pattern even though it is doing nothing. I extended the log-info and hope it's ok

Reply:
Not a big fan of that style, but not strongly against it either. So let it be.